### ISSUE TITLE
Extra % token at end of gammapy-0.20.1-environment.yml

### ISSUE THREAD

**Issue by GernotMaier** [gammapy/gammapy#4146]
**Gammapy version**

installation of 0.20.1

**Bug description**

Extra invisible token `%` at the end of gammapy-0.20.1-environment.yml leads to unexpected system behaviour (e.g. any conda command expected a y/N entry). I've tried this twice and could reproduce the error. Removing the `%` solves the issue.

```
cat gammapy-0.20.1-environment.yml
 # Conda environment for Gammapy version 0.20.1
# Install:    conda env create -f gammapy-0.20.1-environment.yml
# Activate:   conda activate gammapy-0.20.1
# Deactivate: conda deactivate

name: gammapy-0.20.1

channels:
  - conda-forge
  - sherpa

variables:
  PYTHONNOUSERSITE: "1"

dependencies:
  - gammapy=0.20.1
  - python=3.8
  - ipython=8.3
  - cython
  - jupyter
  - jupyterlab=3.4.0
  - pyyaml=6.0
  - click=8.1.3
  - numpy=1.22.3
  - scipy=1.8.0
  - pandas=1.4.2
  - matplotlib=3.5.2
  - astropy=5.0
  - regions=0.6
  - iminuit=2.11.2
  - sherpa=4.14.0
  - healpy=1.15
  - naima=0.10.0
  - pydantic=1.9.0
  - emcee=3.1.1
  - corner=2.2.1
  - pip
  - requests%
  ```

**Expected behavior**

**To Reproduce**

Above issue occurs while working on SL 7.9 with conda 4.12.0 and mamba 0.15.3. Tried is also on OS X without issues.

**Other information**
Any other information you think will be useful for us to fix the issue can go here.

**Comment by adonath** [comment]
Thanks @GernotMaier! I cannot see the percent sign in the source file. I wonder whether it's rather the behavior with the `zsh` shell. See e.g. https://stackoverflow.com/a/44276199/19802442 or https://unix.stackexchange.com/questions/167582/why-zsh-ends-a-line-with-a-highlighted-percent-symbol

The fix might still be just to include a new line at the end in the `environment.yml` file?

**Comment by GernotMaier** [comment]
Yes, I am using `zsh` and it pretty much looks like this is a zshell issue - how annoying is that!

I've tested it and a newline solves the problem, thanks!

(I let you close the issue)
